pkgs/applications/misc/syncthingtray/default.nix

```diff
@@ -0,0 +1,56 @@
+{ mkDerivation
+, stdenv
+, lib
+, fetchFromGitHub
+, qtbase
+, qtwebengine
+, qtdeclarative
+, extra-cmake-modules
+, cpp-utilities
+, qtutilities
+, cmake
+, kio
+, plasma-framework
+, qttools
+, webviewSupport ? false
+, jsSupport ? false
+, enableKioPluginSupport ? false
```

Comment:
The "enable" in the flags is redundant and inconsistent, so better to just use `kioPluginSupport`, `plasmoidSupport`.

Reply:
Makes sense.